src/infrastructure/channels/sequenced_channel.rs

```diff
@@ -0,0 +1,27 @@
+use super::Channel;
+
+use packet::PacketData;
+use infrastructure::DeliveryMethod;
+use error::NetworkResult;
+
+use std::io::Cursor;
+
+pub struct SequencedChannel {
```

Comment:
Docstring please

Reply:
Oke, left it away because it's not used yet. But I'll do it.

Reply:
I think we should start denying missing doc comments on public interfaces.

Reply:
and treat warnings as errors on ci. We can open an issue for this though.